packages/@okta/vuepress-theme-prose/components/LiveWidgetJSCodeMirror.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <textarea disabled ref="codemirrorjs" name="codemirrorjs" id="codemirrorjs"></textarea>
+</template>
+
+<script>
+import 'codemirror/lib/codemirror.css';
+import 'codemirror/theme/neat.css';
+import 'codemirror/addon/lint/lint.css'
+
+import {JSHINT} from 'jshint'
+
+export default {
+  name: 'LiveWidgetJSCodeMirror',
+  props: ['initialConfig'],
+  mounted: async function() {
+    // load side effects for code mirror, linter, matchbrackets, etc. 
+    await import('codemirror/mode/javascript/javascript.js');
+    await import('codemirror/addon/lint/lint')
+    await import('codemirror/addon/lint/javascript-lint')
+    await import('codemirror/addon/edit/matchbrackets')
+    window.JSHINT = JSHINT
```

Comment:
Do we actually call JSHINT somewhere? It appears this is the only place it's referenced.

Reply:
JSHINT is used inside CM linters and validators. Need to put in window object, cos they expect it to be sitting there.